src/running-a-sora-testnet-node.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Running a Node on the SORA v3 Testnet | SORA Docs"
+head:
+  - - meta
+    - name: description
+      content: "Learn how to run a node on the SORA v3 Testnet and participate in the testing of new features."
```

Comment:
```suggestion
      content: "Learn how to run a node on the SORA v3 Testnet and participate in testing new features."
```

Reply:
Resolved in commit 88c29302a00fa18434ec2ba84d3c872148245122